doc/src/manual/faq.md

```diff
@@ -818,10 +818,13 @@ to strings); similarly, `repeat` can be used instead of `^` to repeat strings. T
 
 ### What is the difference between "using" and "import"?
 
-There is only one difference, and on the surface (syntax-wise) it may seem very minor. The difference
-between `using` and `import` is that with `using` you need to say `function Foo.bar(..` to
-extend module Foo's function bar with a new method, but with `import Foo.bar`,
-you only need to say `function bar(...` and it automatically extends module Foo's function bar.
+There are several differences between `using` and `import`
+(see the [Modules section](https://docs.julialang.org/en/v1/manual/modules/#modules)),
+but there is an important difference that may not seem intuitive at first glance,
```

Comment:
perhaps "including in particular one important difference that may not seem" instead of "but there is an important difference that may not seem"

Reply:
Regarding this sentence, where does `using` and `import` comes?

Can you please provide a complete suggestion so I see where the `using` and `import` comes in?